image/base.sh

```diff
@@ -0,0 +1,32 @@
+#!/usr/bin/env bash
```

Comment:
I think it would help with maintainability if you could, for each file we got from phusion, clearly mark which sections are lifted verbatim and which are your own customizations.

That way if we ever need to upgrade or take patches we'll have a fighting chance...